helper.js

```diff
@@ -1,2 +1,7 @@
 
-
+// Generates a 6 digit acess code
+function getAccessCode(){
+    var min = 100000;
+    var max = 999999;
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
```

Comment:
Thanks for doing this, the only thing we need here now is the connection to the other files via module export, but I can add it!